ui/app/components/clients/history.js

```diff
@@ -238,6 +240,8 @@ export default class History extends Component {
         this.responseRangeDiffMessage = null;
       }
     } catch (e) {
+      this.isError = true;
+      this.errorObject = e;
```

Comment:
I think we can get away with having only `errorObject`, set to null to start, and then check for that object in the template so we don't need both of these.